src/dumpulator/modules.py

```diff
@@ -27,13 +28,26 @@ def _parse_pe(self):
         self.entry: int = self.base + self.pe.OPTIONAL_HEADER.AddressOfEntryPoint
         self.pe.parse_data_directories(directories=[pefile.DIRECTORY_ENTRY["IMAGE_DIRECTORY_ENTRY_EXPORT"]])
         pe_exports = self.pe.DIRECTORY_ENTRY_EXPORT.symbols if hasattr(self.pe, "DIRECTORY_ENTRY_EXPORT") else []
+
         for pe_export in pe_exports:
-            va = self.base + pe_export.address
             if pe_export.name:
                 name = pe_export.name.decode("ascii")
             else:
                 name = None
-            export = ModuleExport(va, pe_export.ordinal, name)
+
+            section = self.pe.get_section_by_rva(pe_export.address)
+            section_name = section.Name.rstrip(b"\0").decode()
+
+            # if import points to rdata then export is a forwarder
+            # RtlNtdllName in x64 is the only function that breaks this rule
+            if section_name == ".rdata" and name != "RtlNtdllName":

```

Comment:
The check should be whether the export points inside the export data directory range, that’s how the loader determines this.

Reply:
looking at `pefile` it has already been implemented there